### ISSUE TITLE
Closing a spot websocket connection

### ISSUE THREAD

**Issue by emmanuelkimberley** [jaggedsoft/node-binance-api#798]
Hello,

Thank you for this great package. It has worked great for me so far!
However, I am struggling to find how to close a spot websocket connection. 
I am running node and electron, and the code to connect to the ws is the one provided in the docs. The problem is that the only way for me to stop the connection is to close the app completely.
I have tried to look through your code to check if I could find or add an extra function to easily close a websocket connection but I couldn't manage to solve my issue.

Any help would be greatly appreciated.

let binance = new Binance().options({
  APIKEY: binance_api_key,
  APISECRET: binance_prv_key
});

binance.websockets.candlesticks([`${symbol}`], timeframe, async (candlesticks) => {
  let { e:eventType, E:eventTime, s:symbol, k:ticks } = candlesticks;
  let { o:open, h:high, l:low, c:close, v:volume, n:trades, i:timeframe, x:isFinal, q:quoteVolume, V:buyVolume, Q:quoteBuyVolume } = ticks;})

**Comment by flashup** [comment]
https://github.com/jaggedsoft/node-binance-api/blob/master/examples/advanced.md#terminate-websocket-connections

**Comment by emmanuelkimberley** [comment]
Thank you for the heads up! 
Turns out, even when I set the Binance options to 'reconnect': false, and I used "binance.websockets.terminate('ethbtc@kline_1m')" (with the correct symbol and time frame, of course), It was returning an error 1006, and the stream was continuing. 
I ended up using the ws module for the websocket candlestick streams and it works.

Thanks again for the help though!
